### DISCUSSION TITLE
i18n

### DISCUSSION THREAD

**Discussion by pablohpsilva** [rainbow-me/rainbowkit#586] [Q&A] [answered]
Hello guys, great library!

For my client, it is mandatory to have i18n for a few languages like RU, ES, PT, and EN. Is there a native way to enable this on RainbowKit? 
If not, would it be possible to set the text on my own?
Or the best would be to use the Custom Wallet List and build my own list?

**Comment by DanielSinclair** [comment, marked as answer]
This is now available with [v1.1.0](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.1.0)
